### PR TITLE
chore: add forgotten test after updating the `UpdateFeatureFlag` function.

### PR DESCRIPTION
**Pull request by sashaostrikov**
Test plan:
Unit test added.

### CODE REVIEW

File: internal/database/feature_flags_test.go [hunk 1]
```diff
@@ -726,26 +726,56 @@ func testGetFeatureFlag(t *testing.T) {
...
+		boolFlag.Bool.Value = false
+		updatedFlag, err := flagStore.UpdateFeatureFlag(ctx, boolFlag)
+		require.NoError(t, err)
+		assert.False(t, updatedFlag.Bool.Value)
+		assert.Greater(t, updatedFlag.UpdatedAt, boolFlag.UpdatedAt)
+	})
+	t.Run("rollout flag successful update", func(t *testing.T) {
+		rolloutFlag, err := flagStore.CreateRollout(ctx, "update-test-rollout-flag", 42)
+		require.NoError(t, err)
+		const expectedValue = int32(1337)
```
Comment (indradhanush) [inline internal/database/feature_flags_test.go:774]: :D

File: internal/database/feature_flags_test.go [hunk 2]
```diff
@@ -726,26 +726,56 @@ func testGetFeatureFlag(t *testing.T) {
...
+		require.Nil(t, updatedFf)
+	})
+	t.Run("boolean flag successful update", func(t *testing.T) {
+		boolFlag, err := flagStore.CreateBool(ctx, "update-test-true-flag", true)
+		require.NoError(t, err)
+		boolFlag.Bool.Value = false
+		updatedFlag, err := flagStore.UpdateFeatureFlag(ctx, boolFlag)
+		require.NoError(t, err)
+		assert.False(t, updatedFlag.Bool.Value)
+		assert.Greater(t, updatedFlag.UpdatedAt, boolFlag.UpdatedAt)
```
Comment (indradhanush) [inline internal/database/feature_flags_test.go:769]: Why not use `require.False` ?

**Review by cbart — APPROVED** [review]

**Review by indradhanush — APPROVED** [review]
Personally I try to use only `require` or `assert` but not both at the same time depending on existing precedence in the test module. 

Although I felt `require` has better output than `assert` on failure once and end up almost always using `require` where I can. Not blocking, it comes down to personal preference I guess. :)

**Comment by sashaostrikov** [comment]
@indradhanush 

> Personally I try to use only require or assert but not both at the same time depending on existing precedence in the test module.

> Although I felt require has better output than assert on failure once and end up almost always using require where I can. Not blocking, it comes down to personal preference I guess. :)

I used `require` all the time, but then Coury mentioned in one of the reviews that it's better to use `assert` for non-fatal assertions, which should not interrupt the test run (`require` has a `t.Fatal` under the hood and stops the test) and I agree with him and since then I am using `require` mostly for error checks or other checks which will make the further test run senseless. And I use `assert` for non-critical _assertions_ about expected/actual data so that for example if we have 5 things to check, we can have all of them checked and log the result instead of failing fast with the first unsuccessful check, not having the full picture of what went wrong :)

Hope it makes sense!

**Comment by indradhanush** [comment]
@sashaostrikov Makes sense. TIL too! Thank you for the context! 👍

**Pull request merged**